### PR TITLE
Jcd/register

### LINKED ISSUE #4: registrering
- [x] new html form for registrering 
- [x] style 
- [x] js functionality - noroff mails-

### PR DESCRIPTION
**Pull request by Jean-Christoffer**

### CODE REVIEW

File: html/register/register.html [hunk 1]
```diff
@@ -49,7 +49,7 @@ <h1>Register</h1>
                         </div>
```
Comment (AnaCascante) [inline html/register/register.html:49]: Do we need that here? "Already have a account? Log in"

File: html/register/register.html
Comment (AnaCascante) [inline html/register/register.html:1]: missing the logo - copy paste from log in :)

File: html/register/register.html (hunk 1 above)
Comment (Jean-Christoffer) [reply html/register/register.html:49]: Yes, Else user will have no way back if they misclick or if they sign up for a account they can press login

File: html/register/register.html
Comment (Jean-Christoffer) [reply html/register/register.html:1]: Will fix !

File: html/register/register.html (hunk 1 above)
Comment (AnaCascante) [reply html/register/register.html:49]: got it :). write "an account"  ;)

Comment (AnaCascante) [reply html/register/register.html:49]: good jobb!

Comment (Jean-Christoffer) [reply html/register/register.html:49]: Will do!

**Pull request merged**